k9mail/src/main/java/security/planck/ui/verifypartner/VerifyPartnerViewModel.kt

```diff
@@ -208,7 +208,8 @@ constructor(
         }
 
     private suspend fun refreshRating(): ResultCompat<Unit> =
-        retrieveNewRating().flatMapSuspend { onRatingChanged(it) }
+        if (isMessageIncoming) ResultCompat.success(Unit)
```

Comment:
is this check for is Message Incoming capturing all the needs? What if he accepts on compose of a new msg or on a sent message?

Reply:
yep, because of the behavior change, in the case of incoming message (the message view screen):
- We don't need to retrieve new rating
- We don't need to update the new rating
- We don't nee to save the new rating.
- Sent messages do not allow to do trust management, this is not new.
- Trust management in compose screen (new or replied, forwarded message) has not changed.

I added simple logic for this distinction, as chances are we will have to revert it in the future.

Reply:
ok good to know thanks. It is not consistent with Outlook where Sent msgs allow trust mgmt but if that is ok then good for me, the change looks fine